src/components/CourseForm/CourseForm.jsx

```diff
@@ -1,14 +1,16 @@
-import React, { useState, useEffect, useCallback } from 'react';
```

Comment:
![image](https://user-images.githubusercontent.com/16895020/128605968-663cc8ce-2a12-4617-8bb6-6240f3200f18.png)
After I update course I see the following error. Did you forget to remove edited course from the list?